Dockerfile-centos-7

```diff
@@ -0,0 +1,32 @@
+FROM centos/systemd
+
+ARG FPM_VERSION=1.11.0
+ARG PYTHON_VERSION=3
+ARG DOCKER_WORKDIR=/usr/share/pdagent-integrations
+ENV PYTHON_VERSION ${PYTHON_VERSION}
+ENV container docker
+ENV DOCKER_WORKDIR ${DOCKER_WORKDIR}
+
+RUN yum install -y -q centos-release-scl
+RUN yum install -y -q createrepo
+RUN yum install -y -q gcc
+RUN yum install -y -q gcc-c++
+RUN yum install -y -q kernel-devel
+RUN yum install -y -q make
+RUN yum install -y -q python27-python-pip
+RUN yum install -y -q python3-pip
```

Comment:
nice - this is a good practice - using independent yum commands to make docker layers for each install so that adding new packages (at the end) doesn't invalidate the previous cache.